docs/Announcement-Post.md

```diff
@@ -0,0 +1,269 @@
+    attn = dot_product_attention(HeadDim, (KHeight, KWidth), q, k, v)  # [Batch, Head, Height, Width, HeadDim]
+```
+
+Moreover, named axes have yet another advantage: they make it even easier to scale up to large models. In particular,
+we can use named axes (and Jax's `pjit` sharding) to implement tensor parallelism and FSDP in a way that is easy to use.
+With Levanter and Haliax, FSDP is as simple as specifying an axis (typically the "embed" or "hidden" axis for transformers)
+that you want to keep your parameters and optimizer states sharded along. We then distinguish between two "configurations"
+of your parameters: one for storage (`parameter_axis_mapping`) and one for compute (`compute_axis_mapping`).
+
+For example, here's what your training script might look like:
+
+
+```python
+  def compute_loss(model, data):
+    ...
+
+
+  @named_pjit(axis_resources=trainer.parameter_axis_mapping)
+  def train_step(opt_state, model, data):
+      with hax.axis_mapping(trainer.compute_axis_mapping):
+          loss, grads = jax.value_and_grad(compute_loss)(model, data)
+      updates, opt_state = optimizer.update(grads, opt_state, params=model)
+      model = eqx.apply_updates(model, updates)
+      return loss, model, opt_state
+```
+
+Under the hood, all we do is use Jax's `pjit` operator to shard the parameters and optimizer states along the axis
+you specify. We then use `hax.axis_mapping` to change the preferred mapping for computing the loss and gradients. Jax
+handles most of the heavy lifting, except Linear layers need a bit of handholding, which Haliax provides.
+
+Tensor Parallelism works in a broadly similar manner, with you specifying how "wide" you want model parallelism to be
+and specifying which axes of your model should be sharded. (For a transformer, this would typically be the Head
+and MLP axes.)
+
+In our experiments, we have found that, for models of the sizes that we can reasonably train with our resources, FSDP
+is more than enough to get good scaling, especially on TPU. In practice, with FSDP, we haven't found tensor parallelism
+to be useful for training transformers to at least 20B parameters. (Tensor parallelism is useful for inference, but
+Levanter's focus is on training.)
+
+For more details, please see the [Architectural Overview](XXX) in the Levanter documentation.
+
+#### Bitwise Reproducibility
+
+One of the benefits of Jax is that it offers strong guarantees for reproducibility. In particular, Jax's fine-grained
+control over random number generation makes it easy to ensure bitwise reproducibility, especially when using TPUs.
+Levanter takes advantage of this to offer bitwise reproducibility for training runs, even after preemption. As an example,
+here is a screenshot of a training run being resumed multiple times, even on different TPU pod slices:
+
+![plot showing bitwise reproducibility with four training runs superimposed with the exact same loss curve](figures/bitwise_repro_curve.png)
```

Comment:
I found the highlighted area on the top left bit distracting. I thought you want to highlight that area, but in reality I guess you are not: 
![CleanShot 2023-05-08 at 19 48 13](https://user-images.githubusercontent.com/8097756/236981259-59375b23-d4b3-4efd-a08d-89d004596d5c.png)

Reply:
yeah, sorry laziness